PW_CPV/previous_schemas/qes_240411.xsd

```diff
@@ -0,0 +1,1391 @@
+      <extension base="string">
+        <attribute type="string" name="NAME"/>
+        <attribute type="string" name="VERSION"/>
+      </extension>
+    </simpleContent>
+  </complexType>
+  <complexType name="creatorType">
+    <simpleContent>
+      <extension base="string">
+        <attribute type="string" name="NAME"/>
+        <attribute type="string" name="VERSION"/>
+      </extension>
+    </simpleContent>
+  </complexType>
+  <complexType name="createdType">
+    <simpleContent>
+      <extension base="string">
+        <attribute type="string" name="DATE"/>
+        <attribute type="string" name="TIME"/>
+      </extension>
+    </simpleContent>
+  </complexType>
+  <!-- CONTROL -->
+  <simpleType name="calculationType">
+  	<restriction base="string">
+   <enumeration value="scf"/>
+  	<enumeration value="nscf"/>
+  	<enumeration value="bands"/>
+  	<enumeration value="relax"/>
+  	<enumeration value="vc-relax"/>
+  	<enumeration value="md"/>
+  	<enumeration value="vc-md"/>
+  	</restriction>
+  </simpleType>
+  <simpleType name="controlRestartModeType">
+  	<restriction base="string">
+    <enumeration value="from_scratch"/>
+  	<enumeration value="restart"/>
+  	</restriction>
+  </simpleType>
+  <simpleType name="lowhighType">
+  	<restriction base="string">
+  	  <enumeration value="low"/>
+  	  <enumeration value="high"/>
+      <enumeration value="medium"/>
+      <enumeration value="minimum"/>
+      <enumeration value="debug"/>
+      <enumeration value="none"/>
+      <enumeration value="nowf"/>
+  	</restriction>
+  </simpleType>
+  <!-- INPUT TYPE -->
+  <complexType name="atomic_speciesType">
+    <sequence>
+      <element type="qes:speciesType" name="species" maxOccurs="unbounded"/>
+    </sequence>
+    <attribute type="positiveInteger" name="ntyp"/>
+    <attribute type="string" name = "pseudo_dir" use = "optional"/>
+  </complexType>
+  <complexType name="speciesType">
+    <sequence>
+      <element type="double" name="mass" minOccurs="0" />
+      <element type="string" name="pseudo_file"/>
+      <element type="double" name="starting_magnetization" minOccurs="0" default="0.0"/>
+      <element type="double" name="spin_teta" minOccurs="0" maxOccurs="1" />
+      <element type="double" name="spin_phi"  minOccurs="0" maxOccurs="1" />
+    </sequence>
+    <attribute type="string" name="name"/>
+  </complexType>
+  <complexType name="atomic_structureType">
+    <sequence>
+      <choice minOccurs="1" maxOccurs="1">
+         <element type="qes:atomic_positionsType" name="atomic_positions" minOccurs="0"  />
+         <element type="qes:wyckoff_positionsType" name="wyckoff_positions" minOccurs="0" />
+         <element type="qes:atomic_positionsType" name="crystal_positions" minOccurs="0" />
+      </choice>
+      <element type="qes:cellType" name="cell"/>
+    </sequence>
+    <attribute type="positiveInteger" name="nat" />
+    <attribute type="integer" name = "num_of_atomic_wfc" use="optional"/>
```

Comment:
This one used to be in the `band_structure`, I believe. Why was this moved to the `atomic_structure`? 🤔

Reply:
We initially needed to read `num_of_atomic_wfc` before allocating the data structures for reading band_structure. Instead of moving it, we could put it in both places.